lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -125,6 +132,15 @@ export class PopupClient extends StandardInteractionClient {
 
             if (serverParams.accountId) {
                 this.logger.verbose("Account id found in hash, calling WAM for token");
+                // end measurement for server call with native brokering enabled
+                if (fetchNativeAccIdMeasurement) {
+                    fetchNativeAccIdMeasurement.endMeasurement({
+                        success: true,
+                        isNativeBroker: true
+                    });
+                    fetchNativeAccIdMeasurement.flushMeasurement();
```

Comment:
`flushMeasurement` does not need to be called here.

Reply:
Addressed